ReactAndroid/src/main/java/com/facebook/react/modules/statusbar/StatusBarModule.java

```diff
@@ -40,6 +48,19 @@ public String getName() {
     return "StatusBarManager";
   }
 
+  @Override
+  public @Nullable Map<String, Object> getConstants() {
```

Comment:
You should be able to leave off the `@Nullable` assuming the return types are covariant (think I have that right)